README.md

```
@@ -1,52 +1,140 @@
+    - **Reliability counts**: Minutiae distribution across reliability thresholds (0.5 to 0.9)
+    - **Quality map (qmap) distribution**: Proportion of pixels in each quality level (1=excellent to 4=poor)
+    - **NFIQ Score**: Final quality score (1=highest quality, 5=lowest quality)
+    - **Confidence**: Confidence value for the computed quality score
+
+   **Command-line usage:**
+   ```text
+   java -cp nfiq1.0-<version>.jar;lib\*;test-classes\ org.mosip.nist.nfiq1.test.NfiqApplication "imgfile=<path-to-iso-file>" "logs=0"
+   ```
+
+   Parameters:
+  - `imgfile`: Path to the ISO file containing JP2 or WSQ fingerprint image
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)